host/testdata/integration_pinot_cluster.yaml

```diff
@@ -31,12 +31,13 @@ workerconfig:
   enablereplicator: false
   enableindexer: true
 esconfig:
+  version: "v7"
```

Comment:
This is a must-have, because we have a v7 docker container. If we don't have this, Cadence will assume it needs a v6, and there will be a "ES node not found error".